conf/messages.en

```diff
@@ -268,3 +268,7 @@ schemeMembersList.change.hidden = Change {0}
 schemeMembersList.remove.hidden = Remove {0} from the list
 schemeMembersList.error.required = Select yes if you need to add another member
 schemeMembersList.pagination.label = Showing {0} - {1} of {2} members
+
+removeMemberDetails.title = Are you sure you want to remove {0}?
```

Comment:
We are not meant to have the members name in the title - on the content design doc it says
`Are you sure you want to remove this member`

Reply:
Removed PII from title